src/data/database.js

```diff
@@ -348,7 +348,13 @@ class Database {
 
       if ((!rows) || rows.length < 1) {
         // active participant does not exist, this is an error
-        throw ErrorHandler.Factory.createFSPIOPError(ErrorHandler.Enums.FSPIOPErrorCodes.VALIDATION_ERROR, `Unsupported participant '${participantName}'`)
+        if (participantType === 'PAYEE_DFSP') {
+          throw ErrorHandler.Factory.createFSPIOPError(ErrorHandler.Enums.FSPIOPErrorCodes.DESTINATION_FSP_ERROR, `Unsupported participant '${participantName}'`)
+        } else if (participantType === 'PAYER_DFSP') {
```

Comment:
Would you add there to an enum and use it in the function call as it it used in many places

Reply:
Ok, let me look at cs-shared to see if there is an enum I can use here.